deployment/helm/synopsys-alert/values.yaml

```diff
@@ -54,6 +54,8 @@ postgres:
   databaseName: alertdb
   adminUserName: postgres
   adminPassword: ""
+  dbCredentialSecretName: "" #Secret that contains both the database users username & password
+  dbAdminCredentialSecretName: "" #Secret that contains both the database admin username & password
```

Comment:
Is there a specific format the username and password should be in?

Reply:
The README has a section for creating secrets that explains how to set them up. The db*SecretName parameters here are just for the name of the secret that has the actual credentials